concrete/controllers/panel/dashboard.php

```diff
@@ -82,4 +84,131 @@ public function view()
         $this->set('ui', $ui);
         $this->set('tab', $tab);
     }
+
+    public function checkForMenu($c)
+    {
+        $result = $this->renderActiveSubPanels($c);
+        if (!$result || $result == '') {
+            return null;
+        } else {
+            return 1;
+        }
+    }
```

Comment:
You can shorten this too:

` public function checkForMenu($c)
    {
        return !($result = $this->renderActiveSubPanels($c)) || $result == '' ? null : 1;
    }`

Reply:
That's not much readable, @ramoonlenders
What should this function return? A boolean value that tells if renderActiveSubPanels returns something? If so, why does it return an integer (1) or an empty value (null)?
What I'd do is
```
return (string) $this->renderActiveSubPanels($c) !== '';
``

Reply:
I didn't touch the types, of course it's a discussion worth why returning an integer or null. I didn't look into that much detail. I even missed the double equal sign, which should've been triple equal sign. And yes, if a boolean is returned and wherever this function gets called also checks for true/false, your suggestion is way better

Reply:
@mlocati , whats the cleanest way to return an integer using your method?